### PR TITLE
Spctr 2238 fix css broken while in development mode

### PR DESCRIPTION
**Pull request by abhishekrijal**
### Description
I have added a new plugin function in the `webpack.config.js` file to run sequentially commands to compile SCSS while in watch mode. ( while running `npm run start` ). This will fix the broken CSS and missing icons issue during the block's development.

I have used `npm-run-all` [package](https://www.npmjs.com/package/npm-run-all?activeTab=readme]) in the `devDependencies` to achieve the task runner.

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
Tested locally by developing a test block.
Steps:

- `npm install` or `npm install npm-run-all --save-dev --legacy-peer-deps` - To install the dev dependency.
- `npm run start` -  To watch changes.
- `npm run build` -  Compile build.

### Checklist:
- [ Y] My code is tested
- [ Y] My code follows accessibility standards 
- [ Y] My code has proper inline documentation 
- [ Y] I've included any necessary tests 
- [ Y] I've included developer documentation 
- [ Y] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
 const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
 const path = require( 'path' );
+const { spawn } = require( 'child_process' );
 // const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
+class UAGBRunAdditionalProcess {
+	apply( compiler ) {
+		compiler.hooks.afterEmit.tapAsync( 'UAGBRunAdditionalProcess', ( compilation, callback ) => {
+			spawn( 'npm-run-all', ['--sequential', 'build:sass', 'build:placeholder'], { stdio: 'inherit' } );
```
Comment (AbhijitNage123) [inline webpack.config.js:9]: I guess this only run with Linux OS. I'm using windows and getting following error after npm run start -
```events.js:352
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm-run-all ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn npm-run-all',
  path: 'npm-run-all',
  spawnargs: [ '--sequential', 'build:sass', 'build:placeholder' ]
}
```

Comment (abhishekrijal) [reply webpack.config.js:9]: @AbhijitNage123 I tested it on Mac OS but it should work independently of OS, as it's only the npm command. Did you install `npm install npm-run-all --save-dev --legacy-peer-deps` command before running start ?

By the looks of the error here, it looks like your `npm-run-all` did not install properly.

Please check by installing it, if there's still an issue we can take a call to try resolve it further.

Comment (AbhijitNage123) [reply webpack.config.js:9]: @abhishekrijal Yes I tried that still getting the same error - https://share.getcloudapp.com/GGuJE8DG

Comment (adi3890) [inline webpack.config.js:9]:
```suggestion
			spawn( /^win/.test(process.platform) ? 'npm-run-all.cmd' : 'npm-run-all', ['--sequential', 'build:sass', 'build:placeholder'], { stdio: 'inherit' } );
```

Tried this works for me on windows.

Comment (AbhijitNage123) [reply webpack.config.js:9]: This works for me. Thanks @adi3890

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**